cmd/bosun/expr/expr.go

```diff
@@ -238,6 +240,10 @@ func (r ResultSlice) Len() int           { return len(r) }
 func (r ResultSlice) Swap(i, j int)      { r[i], r[j] = r[j], r[i] }
 func (r ResultSlice) Less(i, j int) bool { return r[i].Value.(Number) > r[j].Value.(Number) }
```

Comment:
I would prefer that. It's obvious what's being sorted on instead of you having to read the code or docs.